docs/pages/blog/meet-joy.md

```diff
@@ -0,0 +1,60 @@
+
+Material UI and Joy UI will definitely share most of the conventions, features, and overall philosophy. This will not only decrease the learning curve for any of you who are willing to try but also because we think of the two containing our take on the ideal developer experience for developing design systems. So, if at any time, Joy UI has something that Material UI doesn't, the catch-up will at some point happen, and the same goes for the other way around.
+
+## What are the new things we're introducing in Joy UI?
+
+Aside from a fresher look and feel for your upcoming project, here are a few features we're planning for the alpha version of Joy UI:
+
+### Global variants
+
+Instead of defining variants at the component level, Joy defines variants at the global level. This sounds bold at first, but the more we experiment the more it starts to make sense. Global variants mostly affect three CSS properties: color, background, and border. What this means is that you'll be able to toggle between the same set of variants in any component, allowing for easier consistency across your app. The four variants are text, outlined, soft, and solid.
+
+### Component integration
+
+We put a lot of effort into designing CSS variables for each component so that they can be seamlessly composed with the least customization.
+One good example is the card component, where the border radius of the card’s children automatically adapts to the card.
+These small details make the component resilient to different scenarios which saves tons of time for manual adjustments.
+Joy does not sacrifice the customization experience in order to have the automatic adjustment.
+You are able to override the style completely via the usual CSS or even adjust the predefined CSS variables from the `sx` prop.
+
+### Perfect dark mode for server-side rendering
+
+Joy provides a simple and effective way to prevent UI flickering when users refresh or re-enter the page after they have toggled dark mode on.
+Moreover, it takes care of the synchronization between browser tabs with a single line of code which allows developers to configure attributes and settings as required.
+
+### Unlimited color schemes
+
+Joy will not only allow for simple toggling between dark and light mode but also provide the right tooling so you can provide your users with multiple color scheme
+options. Within each mode, you'll be able to have as many as you want－powered by CSS variables.
+
+## When will I be able to use it?
+
+A significant part of the core infrastructure for Joy has been made and we've been working in these last couple of months on adding more and more structural components. Once we get to a reasonable amount, and with considerable documentation in place, we plan to release an alpha version, which most likely will happen somewhere around the second semester of 2022.
```

Comment:
```suggestion
A significant part of the core infrastructure for Joy has already been developed, and we've been working for the last couple of months on adding more structural components. Once we get to a reasonable number, and with sufficient documentation, we plan to release an alpha version, most likely during the second half of 2022.
```

What are "structural components"?

> we plan to release an alpha version

We already have:
https://www.npmjs.com/package/@mui/joy